src/plugins/unified_search/public/dataview_picker/change_dataview.styles.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const DATA_VIEW_POPOVER_CONTENT_WIDTH = 280;
+
+export const ChangeDataViewStyles = ({ fullWidth }: { fullWidth?: boolean }) => {
```

Comment:
Yes I think it is an emotion pattern, @cchaos can give more info here :)

Reply:
Good catch on the naming actually, yes, we can lowercase this to 
```suggestion
export const changeDataViewStyles = ({ fullWidth }: { fullWidth?: boolean }) => {
```

Reply:
I am pushing a commit

Reply:
Done https://github.com/elastic/kibana/pull/128401/commits/eca58851b9c261d923c458bd1465dab5c6be2501

Reply:
Should it also be for those where we actually use `css`? 
for example, `src/plugins/unified_search/public/filter_bar/filter_bar.styles.ts`, `src/plugins/unified_search/public/search_bar/search_bar.styles.ts` 

I think so, since this isn't a React component

Reply:
Correct :) Done!